spec/automation/unit/method_validation/vm_migrate_task_complete_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe 'vmmigratetask_complete method' do
+  let(:miq_server)       { EvmSpecHelper.local_miq_server }
+  let(:user)             { FactoryGirl.create(:user_with_group, :userid => 'admin', :email => "lufu@redhat.com") }
```

Comment:
Is there a different FactoryGirl reference for the User that could be used here so you do not have to supply `:userid` and `:email`?  Maybe `:user_with_email` or `:user_with_email_and_group`